airbyte-integrations/connectors/source-recurly/config.sample.json

```diff
@@ -0,0 +1,4 @@
+{
+  "access_token": "<your access token>",
```

Comment:
could you place this in the `sample_configs` directory and update it to match the `spec.json` file?